components/styles.js

```diff
@@ -49,6 +49,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: 5,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
I wanted to make it a square as a default, but maybe it should be something else.